src/prediction_service.cpp

```diff
@@ -126,7 +130,7 @@ grpc::Status ovms::PredictionServiceImpl::Predict(
         status = pipelinePtr->execute(executionContext);
         INCREMENT_IF_ENABLED(pipelinePtr->getMetricReporter().getInferRequestMetric(executionContext, status.ok()));
     } else {
-        status = modelInstance->infer(request, response, modelInstanceUnloadGuard);
+        status = infer(*modelInstance, request, response, modelInstanceUnloadGuard);
```

Comment:
not using ovms::infer here, but using in other places

Reply:
In test files we are not in ovms namespace (the same as in capi.cpp).